pkg/repositories/transformers/task_execution.go

```diff
@@ -129,6 +129,29 @@ func CreateTaskExecutionModel(input CreateTaskExecutionModelInput) (*models.Task
 	return taskExecution, nil
 }
 
+// Returns the unique list of logs across an existing list and the latest list sent in a task execution event update.
+func mergeLogs(existing, latest []*core.TaskLog) []*core.TaskLog {
+	if len(latest) == 0 {
+		return existing
+	}
+	if len(existing) == 0 {
+		return latest
+	}
+	existingSet := make(map[string]*core.TaskLog, len(existing))
+	for _, existingLog := range existing {
+		existingSet[existingLog.Uri] = existingLog
+	}
+	// Copy over the existing logs from the input list so we preserve the original order.
+	logs := append([]*core.TaskLog(nil), existing...)
```

Comment:
I meant I think this can be:
```suggestion
	logs := existing
```